pkg/volume/azure_dd/common.go

```diff
@@ -0,0 +1,316 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azure_dd
+
+import (
+	"fmt"
+	"hash/crc32"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path"
+	"regexp"
+	"strconv"
+	STRINGS "strings"
```

Comment:
because "k8s.io/kubernetes/pkg/util/strings" is also imported (for using sets). and standard strings is used for stuff like split() and ToUpper